eduNEXT/xblock-imagesgallery
# feat: delete multiple files at once

### Description

This PR upgrades the delete_file endpoint to accept a list of asset keys to delete

### Testing instructions

1. Go to studio and edit the componente
2. Add some images
![image](https://github.com/eduNEXT/xblock-imagesgallery/assets/33465240/d3ef99ce-d170-4138-b956-7741e09f6349)
3. Click on the delete button hovering the images
4. Verify the image is deleted
![image](https://github.com/eduNEXT/xblock-imagesgallery/assets/33465240/d93a5d14-1f12-4e99-b34f-6daa285cae41)



**Merge checklist:**
Check off if complete *or* not applicable:



**Merge checklist:**
Check off if complete *or* not applicable:
